mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,235 @@
+    self_link: "projects/{{project}}/locations/{{location}}/services/{{service_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      A managed metastore service that serves metadata queries.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 40
+          update_minutes: 40
+          delete_minutes: 40
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'serviceId'
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the metastore service. The id must contain only letters (a-z, A-Z), numbers (0-9), underscores (_),
+          and hyphens (-). Cannot begin or end with underscore or hyphen. Must consist of between
+          3 and 63 characters.
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        url_param_only: true
+        input: true
+        default_value: global
+        description: |
+          The  location where the autoscaling policy should reside.
+          The default value is `global`.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The relative resource name of the metastore service.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: |
+          Time the AccessPolicy was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: |
+          Time the AccessPolicy was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: 'User-defined labels for the metastore service.'
+      # This is an x-product resource reference.
+      - !ruby/object:Api::Type::String
+        name: 'network'
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: 'endpointUri'
+        output: true
+        description: |
+          The URI of the endpoint used to access the metastore service.
+      - !ruby/object:Api::Type::Integer
+        name: 'port'
+        description: |
+          The TCP port at which the metastore service is reached. Default: 9083.
+        default_value: 9083
```

Comment:
Is this a value that we should encode into a default? Or let the API default for us

Reply:
let me try